### DISCUSSION TITLE
Error in Backup and Restore: Initial backup file does not exist

### DISCUSSION THREAD

**Discussion by COUPON4Vitamins** [linuxdeepin/developer-center#3546] [Troubleshooting | 故障排查]
When I navigate to **System Info > Backup and Restore** I see no restore and backup options, only an error message in red. 

![image](https://user-images.githubusercontent.com/20388814/212252988-6af32253-3984-46a8-86fc-0b4abe3c29d0.png)

In the boot menu, I see the option where I can roll back to a previous version of Deepin. Yet, I'm afraid if it works as the Control Center outputs an error.

**Comment by BLumia** [comment]
Which deepin version are you using? deepin v20.8?

**Reply by COUPON4Vitamins** [reply]
Version 20.8, sorry forgot to add
